src/mococrw/openssl_wrap.h

```diff
@@ -75,6 +75,25 @@ struct SSLDeleter
     }
 };
 
+/**
+ * Like SSLDeleter, but takes into account the return type
+ * in the definitions of the "_free" functions.
+ *
+ * We could have modified SSLDeleter to take arbitrary return
```

Comment:
Did you clarified with @garw if we want to increase the minor when adding HSM/PKCS#11 support?

Reply:
Yes.

Reply:
It was preferred to introduce an alternative version of SSLDeleter: https://github.com/bmwcarit/MoCOCrW/pull/106#discussion_r907336017